package.json

```diff
@@ -25,9 +25,7 @@
   },
```

Comment:
please update this and other new packages to be scoped to `@digitalbazaar`.

Reply:
Done.